include/vara/Feature/Feature.h

```diff
@@ -159,6 +159,9 @@ class Feature {
   [[nodiscard]] FeatureSourceRange *getFeatureSourceRange() {
     return Source.has_value() ? &Source.value() : nullptr;
   }
+  void setFeatureSourceRange(FeatureSourceRange &Fsr) {
```

Comment:
```suggestion
  void setFeatureSourceRange(FeatureSourceRange FeatureSR) {
```
If your going to move the object afterwards anyhow taking it as a value is prefered.